components/Modal/Action.vue

```diff
@@ -414,9 +413,29 @@ export default {
         default:
       }
     },
+    setNetworkFees(type) {
+      this.gasEstimateMultiplier = this.setGasMultiplier(type)
+      const defaultNetworkFees = fees.getFees(this.transactionData.type)
+      const networkFees = [
+        {
+          amount: BigNumber(defaultNetworkFees.feeOptions[0].amount)
+            .multipliedBy(this.gasEstimateMultiplier)
+            .toString(),
+          denom: this.feeDenom,
+        },
+      ]
+      return networkFees
+    },
+    setGasMultiplier(type) {
```

Comment:
This function is not setting/changing any variable, I don't suggest naming the function that starts with `set`, perhaps using `get`/`calculate` is better